src/classes/RD_RecurringDonations.cls

```diff
@@ -436,7 +433,8 @@ public class RD_RecurringDonations {
                     *********************************/
                                         
                     OppCloseDate = getNextDate(OppCloseDate, r);
-                
+
+                    // TODO: Rewrite this to use the NPSP settings for Opportunity Naming
```

Comment:
We haven't used TODO's in code, and I strongly discourage them.  If it something that needs to get done, we need to either address it now, or more typically, add it to the backlog.  In this case, I'm not even sure that Opp naming handles RD's where we want N of M included in the name.